src/sentry/tasks/on_demand_metrics.py

```diff
@@ -252,15 +252,7 @@ def _get_widget_on_demand_specs(
         return []
 
     widget_specs = convert_widget_query_to_metric(project_for_query, widget_query, True)
-
-    unique_specs = []
-    hashes = set()
-    for hashed_metric_spec in widget_specs:
-        if hashed_metric_spec[0] not in hashes:
-            unique_specs.append(hashed_metric_spec)
-            hashes.add(hashed_metric_spec[0])
-
-    return unique_specs
```

Comment:
We don't need this check anymore since we're already doing it here: https://github.com/getsentry/sentry/pull/64212/files